src/main/java/io/vlingo/http/resource/sse/SseStreamResource.java

```diff
@@ -109,6 +109,7 @@ public SsePublisherActor(final String streamName, final Class<? extends Actor> f
       this.subscribers = new HashMap<>();
 
       final ActorInstantiator<?> instantiator = ActorInstantiatorRegistry.instantiatorFor(feedClass);
+      if(instantiator==null)throw new IllegalArgumentException("No ActorInstantiator registred for feedClass="+feedClass.toString());
```

Comment:
Since this is an Actor, throwing an exception doesn't help much. However, since this kind of problem will be at the toolset development level will will have some meaning. Generally a toolkit actor should deal with errors a bit more conservatively by making a safe adjustment locally. Throwing an exception in such an actor may otherwise have undesirable consequences. You should read about actor supervision in docs.vlingo.io.